maps_dashboards/common/index.ts

```diff
@@ -29,3 +29,11 @@ export const LAYER_VISIBILITY = {
   NONE: 'none',
   VISIBLE: 'visible',
 };
+
+export interface MapSavedObjectAttributes {
```

Comment:
can we add documentation on top of this interface. and also for the attributes,

Reply:
This is interface is an attribute interface, can you point me to the MapSavedObject?

Reply:
Sure, will add doc on top of it. [Here](https://github.com/opensearch-project/dashboards-maps/pull/67/files#diff-4d9cd2e1a68cb20e9506bc77d26af2fc62906318d5b09a30bd034dc45449559fR8) is the MapSavedObject.

Reply:
Thanks @junqiu-lei for the link, but I cannot see the attributes interface as a parameter of the savedObject? am I missing something?

Reply:
The MapSavedObjectAttributes is mainly used to restrict the type when call savedObjectsClient like [here](https://github.com/opensearch-project/dashboards-maps/pull/67/files#diff-e7e14756f682ecb521f75f21a87dbfc518f0c4d8d40c88338e6513cc8ade2589R46)